skydb/skydb.go

```diff
@@ -88,6 +87,30 @@ func (db SkyDB) Write(data []byte, dataKey crypto.Hash, rev uint64) error {
 	return nil
 }
 
+// EntropyFromEnv returns the configured value of the SKYDB_ENTROPY environment
+// variable or an error.
+func EntropyFromEnv() ([32]byte, error) {
+	var e [32]byte
+	eStr := os.Getenv("SKYDB_ENTROPY")
+	if eStr == "" {
+		return e, errors.New("missing or empty SKYDB_ENTROPY environment variable. it needs to contain 32 bytes of entropy.")
```

Comment:
If you want to distinguish between a var that wasn't set and one that was explicitly set to an empty value, you could use `os.LookupEnv`. Although I don't think that it would provide any additional value in this case.

Also it might be a bit more helpful to say "32 bytes of base64 encoded entropy".

Reply:
"32 bytes of base64 encoded entropy" sounds better, I'll use that.

Regarding the empty vs unset - it's not really important. What I really want to see is 32 bytes of entropy. This message should be enough for the operator to figure out they need to set the env var in their `docker-compose` file.

If you think that adding even more specificity to the error message would make it clearer and more helpful I don't mind adding that but personally I'm not sure it would add any value.